### DISCUSSION TITLE
не работает сайт указанный в list-general.bat

### DISCUSSION THREAD

**Discussion by wontyy** [Flowseal/zapret-discord-youtube#1066] [General]
добавил сайт escapefromtarkov.com и ничего не происходит.Пишет что выполнил 36 хостов(что было изначально при установке),хотя написано 40. 
![image](https://github.com/user-attachments/assets/b85c5888-3fe5-422d-8623-dc03e43978f1)

**Comment by Flowseal** [comment]
Во первых - файл не сохранён
![image](https://github.com/user-attachments/assets/bd2fc41c-79fe-4051-8505-8f3d00062994)

Во вторых - не нужно писать сабдомены.
Написание домена escapefromtarkov.com уже включает
prod.escapefromtarkov.com
abc.escapefromtarkov.com
и так далее

Если не заработает - пробуйте другие батники
